### ISSUE TITLE
Inventory not loading

### ISSUE THREAD

**Issue by Toyroom** [dasilva333/TowerGhostForDestiny#682]
Hi

I was having problems with TGFD on my iPad not properly displaying items, there was a placeholder but no icon on random items in my inventory so I un and reinstalled the app in the hope of rectifying it but since doing so none of my inventory loads.  I have tried un and reinstalling several times plus rebooting my iPad but no joy.  I also tried landscape and portrait mode as have previously seen an issue like that but still nothing.  My inventory loads perfectly on my windows phone app.

All the best, 

Regards, Alan.

**Comment by Toyroom** [comment]
Hi

Just seen the following error message after ticking "Auto Updates", "Problem checking for updates, new manifest timeout".

**Comment by Celevas** [comment]
That's the same problem I was having on my iPad (IOS 10.1.1, TGD 3.9.3.3 IIRC) yesterday.  Plus, every time I would restart TGD, I got the same error message: "Problem checking for updates: new manifest timeout".  In addition, as OP stated, some slots were randomly blank, even though if you tapped on it it would show the name of the item at the bottom.  So after closing and restarting TGD a few times with the same results, after the 3rd or 4th restart, nothing would come up at all even though login appeared to be successful (I never got a log-in error message); all that came up was the background. And from then on every time I restarted it all it would do was log in, give me the new manifest timeout error message and the blank background wallpaper (the same as what it looks like right after it logs in but a few seconds before it loads the items, only it never loads any items, even after a few minutes).  So I deleted TGD from my iPad and installed it again from the App Store (TGD 3.9.0.0), and it too gives me the same error message and is unable to update itself. Specifically, it is unable to even check for updates. And again, no slots even appear, blank or not.  Just the empty background.  I hope this helps narrow down the issue.  BTW, I switched back and forth between my wifi router at home and cellular data on my iPad, but it made no difference.
![img_4647](https://cloud.githubusercontent.com/assets/23483359/20860510/840a9b72-b93f-11e6-9b6e-2f670baee356.PNG)

**Comment by Toyroom** [comment]
Hi

Further to the above, I have now tried it on a PC using the Chrome extension and that also has similar issues except the Vault is displaying ok but nothing for the characters inventories.

A friend has confirmed that it is fine on his Android phone.

Regards

Alan.

**Comment by tonyNOS** [comment]
Same here.

**Comment by hoover2404** [comment]
I'm having the exact same problem as Celevas for several days now.

**Comment by Shaken-K1W1** [comment]
+1 to the update manifest timeout and nothing loading issues. 
Best of luck and definitely the best app I've used for destiny

**Comment by Toyroom** [comment]
Just loaded TGFD on the off chance and an update downloaded and all is back to normal for me now.  Thank you so much, it has felt like I have been playing with one hand tied behind my back 😀

**Comment by Celevas** [comment]
Same here.  I learned about so many of the Destiny Companion's limitations by being forced to use it while TGD was down.  I always took for granted that I could transfer weapons & gear from any character at any time, even if the item is equipped.  Companion can't transfer an item that's equipped, as I recently discovered.  Just one example of many.  When TGD is running, none of the other apps for Destiny even come close.  I don't just love this app; I need it.  Thanks for all you do to keep it going.
